pkg/cmd/cli/cmd/env.go

```diff
@@ -38,18 +39,19 @@ syntax.`
   // List the environment variables defined on all pods
   $ %[1]s env pods --all --list
 
-  // Output a YAML object with updated enviroment for deployment 'registry'
+  // Output a YAML object with updated environment for deployment 'registry'
   // Does not alter the object on the server
```

Comment:
Outputs the modified deployment config, and does not alter the object on the server